package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "fd-slicer",
+  "version": "1.1.0",
+  "description": "safely create multiple ReadStream or WriteStream objects from the same file descriptor",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --reporter spec --check-leaks",
+    "test-cov": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter dot --check-leaks test/index.spec.ts",
+    "test-travis": "istanbul cover node_modules/mocha/bin/_mocha --report lcovonly -- --timeout 10000 --reporter spec --check-leaks test/index.spec.ts"
+  },
+  "author": "Andrew Kelley <superjoe30@gmail.com>",
```

Comment:
can you replace this with your own credentials?

Reply:
I added the maintainer's field instead of changing the author.

I think it worth to keep the original author.

Reply:
makes sense!